Dockerfile

```
@@ -1,23 +1,13 @@
+
+WORKDIR /app
 
-FROM python:3.12.9-alpine as builder
-# create virtual environment
-RUN python -m venv /opt/venv --system-site-packages
-ENV PATH="/opt/venv/bin:$PATH"
 COPY requirements.txt .
-RUN pip install --upgrade pip
-RUN pip install --no-cache-dir -r requirements.txt
 
+RUN pip install --upgrade pip && \
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)